Build.PL

```diff
@@ -24,6 +24,7 @@ my $builder = Module::Build->new(
         'JSON::XS'           => 0,
         'MIME::Lite'         => 0,
         'Net::SMTP::SSL'     => 0,
+        'Text::CSV'          => 0,

```

Comment:
I might be missing something, but if we add `Text::CSV` to the `build_requires` section of Build.PL, Travis will install this module as a dependency. So you don't need to skip the test. For it to pass, one has to have `Text::CSV` installed, otherwise the test will legitimately fail.